control-plane/rest/service/src/v0/swagger_ui.rs

```diff
@@ -34,10 +34,17 @@ fn get_swagger_html(spec_uri: &str) -> Result<String, String> {
 #[derive(Clone)]
 struct GetSwaggerUi(Result<String, String>);
 
-impl Handler<(), Ready<Result<HttpResponse, Error>>> for GetSwaggerUi {
+impl Handler<()> for GetSwaggerUi {
+    type Output = Result<HttpResponse, Error>;
+    type Future = Ready<Result<HttpResponse, Error>>;
+
     fn call(&self, _: ()) -> Ready<Result<HttpResponse, Error>> {
```

Comment:
You can now change the return type to `Ready<Self::Output>`

Reply:
Ah yes, Future can be that and then the result can just be Self::Future